app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java

```diff
@@ -47,11 +46,11 @@
 
     private Adapter adapter;
 
-    public static ContributorsFragment newInstance() {
-        return new ContributorsFragment();
+    public ContributorsFragment() {
     }
 
-    public ContributorsFragment() {
+    public static ContributorsFragment newInstance() {
```

Comment:
OK. I will fix it!

As for the question, is there a code style definition somewhere?
I refer to "codeStyleSettings.xml", but when I use the code format the constructor and static method will be swapped.